src/components/layout/Footer/index.js

```diff
@@ -1,73 +1,25 @@
 import React from 'react'
 
-import { css } from 'styled-components'
-import {
-  FooterWrapper,
-  HeadingSection,
-  SitemapSection,
-  ContactSection,
-  EmailLink,
-  AddressesBlock,
-} from './styles'
-import { H1, H4, CapsHeadingSmall, XSP } from '../../elements'
+import { FooterWrapper, ContentWrapper, EmailLink, Copyright } from './styles'
+import { H2 } from '../../elements'
 
 const Footer = () => (
-  <FooterWrapper>
-    <HeadingSection>
-      <H1>Get in touch!</H1>
-    </HeadingSection>
-    <SitemapSection>
-      {/* todo: make the sitemap items as Gatsby links */}
-      <CapsHeadingSmall
-        css={css`
-          margin-bottom: 2rem;
-        `}
-      >
-        sitemap
-      </CapsHeadingSmall>
-      <H4>
-        <span>1.0</span>Our Approach + Services
-      </H4>
-      <H4>
-        <span>2.0</span>Our Work + Case Studies
-      </H4>
-      <H4>
-        <span>3.0</span>Our Story + Team
-      </H4>
-      <H4>
-        <span>4.0</span>Get in touch
-      </H4>
-    </SitemapSection>
-    <ContactSection>
-      <CapsHeadingSmall>contact</CapsHeadingSmall>
-      <EmailLink
-        href="mailto:hello@yallacooperative.com?Subject=Hello%20again"
-        target="_top"
-      >
-        hello@yallacooperative.com
-      </EmailLink>
-      <AddressesBlock>
-        <div>
-          <CapsHeadingSmall>london</CapsHeadingSmall>
-          <XSP>
-            149 Fonthill Rd <br /> Finsbury Park <br /> London <br /> N4 3HF
-          </XSP>
-        </div>
-        <div>
-          <CapsHeadingSmall>germany</CapsHeadingSmall>
-          <XSP>
-            149 Fonthill Rd <br /> Finsbury Park <br /> London <br /> N4 3HF
-          </XSP>
-        </div>
-        <div>
-          <CapsHeadingSmall>gaza</CapsHeadingSmall>
-          <XSP>
-            149 Fonthill Rd <br /> Finsbury Park <br /> London <br /> N4 3HF
-          </XSP>
-        </div>
-      </AddressesBlock>
-    </ContactSection>
-  </FooterWrapper>
+  <>
+    <FooterWrapper>
+      <ContentWrapper>
+        <H2>Get in touch!</H2>
+        <EmailLink
+          href="mailto:hello@yallacooperative.com?Subject=Hello%20again"
+          target="_top"
+        >
+          hello@yallacooperative.com
+        </EmailLink>
+      </ContentWrapper>
+    </FooterWrapper>
+    <Copyright>
+      <p>© 2020 Yalla Cooperative</p>
+    </Copyright>
+  </>
```

Comment:
is this how the footer looks now? the design changed?

Reply:
For now yeh as we don't have proper content for the addresses etc and the different links but once everything's done planning to change it back to the original